ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -6,13 +6,31 @@
 
 @to_ivy_arrays_and_back
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
-def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
-    return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
-
+def batch_norm(
+    x,
+    mean,
+    variance,
+    offset=None,
+    scale=None,
+    training=False,
+    epsilon=1e-5,
+    momentum=1e-1,
+    data_format="NSC",
+):
```

Comment:
The frontend function should have the same args as the native paddle batch_norm function 
```
paddle.nn.functional.batch_norm(x, running_mean, running_var, weight, bias, training=False, momentum=0.9, epsilon=1e-05, data_format='NCHW', use_global_stats=None, name=None)
```